README.md

```diff
@@ -1,17 +1,78 @@
-# How to use Open Virtual Networking with Kubernetes
+# Open Virtual Networking (OVN)
 
-This document describes how to use Open Virtual Networking with Kubernetes
+OVN provides network virtualization to containers.  In the "overlay" mode,
+OVN can create a logical network amongst containers running on multiple hosts.
+In this mode, OVN programs the Open vSwitch instances running inside your
+hosts.  These hosts can be bare-metal machines or vanilla VMs.
+
+## OVN via kubernetes daemonsets
+
+The easiest way to use OVN as the networking plugin (via CNI) for kubernetes
+is via kubernetes daemonsets. This currently only works for Linux nodes.
+
+One simply needs to apply the following yamls.
+
+1. Create a namespace `ovn-kubernetes` in kubernetes
+
+`kubectl create -f https://github.com/openvswitch/ovn-kubernetes/raw/master/dist/yaml/ovn-namespace.yaml`
+
+2. Create service accounts and policies for OVN
+
+`kubectl create -f https://github.com/openvswitch/ovn-kubernetes/raw/master/dist/yaml/ovn-policy.yaml`
+
+3. Run a config map file for OVN.
+
+* $MASTER_IP below is the IP address of the
+kubernetes master node.
```

Comment:
$MASTER_IP is the ip address of the OVN master node (which is usually the cluster master). Note there can and must be one OVN master node. (Multiple master nodes is not currently supported).